Sources/StreamChat/WebSocketClient/EventMiddlewares/TypingStartCleanupMiddleware.swift

```diff
@@ -14,8 +14,6 @@ class TypingStartCleanupMiddleware: EventMiddleware {
     /// A closure that will be invoked with `stop typing` event when the `incomingTypingStartEventTimeout` has passed
     /// after `start typing` event.
     let emitEvent: (Event) -> Void
-    /// A closure to get a list of user ids to skip typing events for them.
-    let excludedUserIds: () -> Set<UserId>
```

Comment:
Why do we remove this?

Reply:
This was on of the responsible for the circular dependency. We were using the `authenticationRepository` to get the current user, and populate here the `excludedUserIds`. This is over complicated for something that is really simple todo. We have access to the current user in the `DatabaseContainer` inside this `Middleware`. So we have simplified the code, and fixed the circular dependency 👍